nautobot/circuits/templates/circuits/inc/circuit_termination.html

```diff
@@ -119,6 +119,37 @@
                 <td>Description</td>
                 <td>{{ termination.description|placeholder }}</td>
             </tr>
+            {% with relationships=termination.get_relationships_data %}
+                {% if relationships.source or relationships.destination or relationships.peer %}
+                    {% for side, relationships in relationships.items %}
+                        {% for relationship, value in relationships.items %}
+                            <tr>
+                                <td><span title="{{ relationship.slug }}">{{ value.label }}</span></td>
+                            {% if not value.has_many and value.value %}
+                                <td>
+                                    <a href="{{ value.url }}">{{ value.value }}</a>
+                                </td>
+                            {% elif value.has_many and value.queryset.count %}
+                                <td>
+                                    <a href="{% url 'extras:relationshipassociation_list' %}?relationship={{relationship.slug}}&{{side}}_id={{termination.id}}">
+                                        {{ value.queryset.count }}
+                                        {% if value.queryset.count > 1 %}
+                                            {{ value.peer_type.model_class|meta:"verbose_name_plural" }}
+                                        {% else %}
+                                            {{ value.peer_type.model_class|meta:"verbose_name" }}
+                                        {% endif %}
+                                    </a>
+                                </td>
+                            {% else %}
+                                <td>
+                                    <span class="text-muted">&mdash;</span>
+                                </td>
+                            {% endif %}
+                            </tr>
+                        {% endfor %}
+                    {% endfor %}
```

Comment:
It might be possible/desirable to extract this block into another `inc` template (`core/templates/inc/relationships_table_rows.html` or something like that?) that could be reused across both `circuits/inc/circuit_termination.html` and `templates/inc/relationships_panel.html` to reduce duplication - what do you think?

Reply:
The thought did occur to me! OK I'll do that then!